src/pudl/transform/params/ferc1.py

```diff
@@ -2968,17 +2974,157 @@
+                    "utility_plant_net",
+                    "amortization_and_depletion_of_producing_natural_gas_land_and_land_rightsutility_plant_in_service",
+                    "amortization_utility_plant_held_for_future_use",
+                    "amortization_and_depletion_utility_plant_leased_to_others",
+                    "accumulated_provision_for_depreciation_amortization_and_depletion_of_plant_utility",
+                    "utility_plant_in_service_property_under_capital_leases",
+                    "utility_plant_acquisition_adjustment",
+                    "utility_plant_leased_to_others",
+                    "utility_plant_held_for_future_use",
+                    "amortization_of_other_utility_plant_utility_plant_in_service",
+                    "amortization_of_underground_storage_land_and_land_rightsutility_plant_in_service",
+                    "utility_plant_in_service_completed_construction_not_classified",
+                    "utility_plant_in_service_plant_purchased_or_sold",
+                    "construction_work_in_progress",
+                    "utility_plant_in_service_experimental_plant_unclassified",
+                ]
+            }
+        },
+        "drop_invalid_rows": [
+            {
+                "invalid_values": [pd.NA, np.nan, ""],
+                "required_valid_cols": ["utility_plant_value"],
+            },
+        ],
+        "wide_to_tidy": {
+            "xbrl": {
+                "idx_cols": ["entity_id", "report_year", "utility_type_axis"],
+                "value_types": ["utility_plant_value"],
+                "expected_drop_cols": 1,
+                "stacked_column_name": "xbrl_factoid",
+            },
+            "dbf": {
+                "idx_cols": [
+                    "report_year",
+                    "record_id",
+                    "utility_id_ferc1",
+                    "utility_type",
+                    "utility_type_other",
+                ],
+                "value_types": ["utility_plant_value"],
+                "expected_drop_cols": 1,
+                "stacked_column_name": "utility_plant_asset_type",
+            },
+        },
+        "merge_metadata_xbrl": {
+            "rename_columns": {"xbrl_factoid": "utility_plant_asset_type"},
+            "on": "utility_plant_asset_type",
+        },
+        "align_row_numbers_dbf": {"dbf_table_name": "f1_utltyplnt_smmry"},
+        "categorize_strings": {
+            "utility_type": {
+                "categories": {
+                    "total": ["total"],
+                    "electric": ["electric", "ferc:ElectricUtilityMember"],
+                    "gas": ["gas", "ferc:GasUtilityMember"],
+                    "common": ["common", "ferc:CommonUtilityMember"],
+                    "other1": ["other1", "ferc:OtherUtilityMember"],
+                    "other2": ["other2", "ferc:OtherUtility2Member"],
+                    "other3": ["other3", "ferc:OtherUtility3Member"],
```

Comment:
Probably not important for the moment, but I'm sure we'll see more of these categories in other table, and when they show up we'll probably want to pull this out into a re-usable component somehow.

Reply:
yea agreed